src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -55,7 +55,7 @@ public function finishView(FormView $view, FormInterface $form, array $options):
         $attr = $view->vars['attr'] ?? [];
 
         $controllerName = 'symfony--ux-autocomplete--autocomplete';
-        $attr['data-controller'] = trim(($attr['data-controller'] ?? '').' '.$controllerName);
+        $attr['data-controller'] = $controllerName.' '.trim($attr['data-controller'] ?? '');
```

Comment:
```suggestion
        $attr['data-controller'] = trim($controllerName.' '.($attr['data-controller'] ?? ''));
```

Reply:
I'm not sure here.. as i believe we recommand the other way around for now..

https://symfony.com/bundles/ux-autocomplete/current/index.html#extending-tom-select